transformer_engine/pytorch/attention.py

```diff
@@ -5267,15 +5272,108 @@ def forward(
             query_layer.shape[0],
             key_layer.shape[0],
         )
-
-        attn_mask_type, attention_mask, actual_seqlens_q, actual_seqlens_kv = get_full_mask(
-            max_seqlen_q,
-            max_seqlen_kv,
-            attn_mask_type=attn_mask_type,
-            attention_mask=attention_mask,
-            window_size=window_size,
-            attention_type=self.attention_type,
-        )
```

Comment:
I'm still not sure why it's picking up these changes as these are not present in your `paged_attention` branch. @cyanguwa

Reply:
My bad. These changes are in `UnfusedDPA`. Let me remove them as I don't think we need to use them